tests/modules/nf-scil/preproc/n4/test.yml

```diff
@@ -0,0 +1,9 @@
+- name: "preproc n4"
+  command: nextflow run ./tests/modules/nf-scil/preproc/n4 -entry test_preproc_n4 -c ./tests/config/nextflow.config
+  tags:
+    - "preproc"
+    - "preproc/n4"
+  files:
+    - path: "output/preproc/test_dwi_n4.nii.gz"
```

Comment:
Did you generate the md5sum yourself ? Nextflow does not like it on my end when running the tests. If yes, just remove it, I will regen all md5sum anyway when we have the true test infra.